server/sonar-server/src/main/java/org/sonar/server/qualitygate/QualityGateFinder.java

```diff
@@ -72,6 +72,7 @@ public QGateWithOrgDto getByOrganizationAndId(DbSession dbSession, OrganizationD
       "No quality gate has been found for id %s in organization %s", qualityGateId, organization.getName());
   }
 
+  // TODO As there is always a default quality gate, this method should not return an optional
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)